ui/app/AppLayouts/Communities/popups/InviteFriendsToCommunityPopup.qml

```diff
@@ -96,7 +96,7 @@ StatusStackModal {
         ProfilePopupInviteFriendsPanel {
             rootStore: root.rootStore
             contactsStore: root.contactsStore
-            community: root.community
+            communityId: root.communityId
```

Comment:
```suggestion
            communityId: root.community.id
```

Reply:
This is wrong, and breaks the invitation to the community

Reply:
I changed the signature to get `communityId` instead of the whole community. It made no sense to pass the whole object if all we needed was the ID.
```qml
    property string communityId
```